egui_glow/src/painter.rs

```diff
@@ -268,16 +195,26 @@ impl Painter {
         if self.egui_texture_version == Some(texture.version) {
             return; // No change
         }
-
+        let gamma = if self.post_process.is_none() {
+            1.0 / 2.0
```

Comment:
Likely `1.0 / 2.2` will look slightly better (better approximates the sRGB gamma curve)

Reply:
sorry i miss typed it.